source/pdk/1.0/pdk_install.md

```diff
@@ -81,16 +78,16 @@ If you are using PDK behind a proxy, you must set environment variables before r
 
 You can either set these variables on the command line before each working session or add them to your system configuration, which varies depending on the operating system.
 
-On Windows systems, run:
+On Linux-based systems or OS X, run:
```

Comment:
Haha, I argued with the devs about this arrangement myself (I wanted it Linux first too).  I am vindicated.